loopingStatements/index.js

```diff
@@ -0,0 +1,127 @@
+function forinloop(){ 
+
+            const employee={name:"anu",location:"chennai",experience:10}; 
+
+            let val=" "; 
+
+            for(let x in employee){ 
```

Comment:
How will you use it for nested objects like { employee: {name:"anu",location:"chennai",experience:10}}